manage_fastapi/templates/project/{{ cookiecutter.folder_name }}/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV PYTHONPATH "${PYTHONPATH}:/"
 ENV PORT=8000
 {% if cookiecutter.packaging == "poetry" %}
 # Install Poetry
-RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_HOME=/opt/poetry python && \
+RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/poetry python && \
     cd /usr/local/bin && \
     ln -s /opt/poetry/bin/poetry && \
```

Comment:
New installer changes the way Poetry is installed (it's similar to how pipx does it), so this probably won't work as intended

Reply:
You mean on new Apple chips also ?

Reply:
The installer is cross-platform